ClassicAssist/UO/Gumps/MacrosGump.cs

```diff
@@ -35,19 +35,27 @@ public class MacrosGump : RepositionableGump
         private static IEnumerable<MacroEntry> _lastMacros;
         private readonly MacroEntry[] _macros;
 
-        public MacrosGump( IEnumerable<MacroEntry> macros ) : base( 190, 180, _serial++, (uint) _serial++ )
+        // TODO : read value from options tab
+        //private static int _sizeX = Options.CurrentOptions.MacrosGumpHeight;
+        //private static int _sizeY = Options.CurrentOptions.MacrosGumpWidth;
+
+        public MacrosGump( IEnumerable<MacroEntry> macros ) : base( Options.CurrentOptions.MacrosGumpWidth, Options.CurrentOptions.MacrosGumpHeight, _serial++, (uint) _serial++ )
         {
             _macros = macros.ToArray();
 
             GumpX = Options.CurrentOptions.MacrosGumpX;
             GumpY = Options.CurrentOptions.MacrosGumpY;
 
-            Movable = false;
-            Closable = false;
+            int _height = Options.CurrentOptions.MacrosGumpHeight;
+            int _width = Options.CurrentOptions.MacrosGumpWidth;
+
+            Movable = true;
```

Comment:
Set Movable and Closable back to false, if they move the gump around by dragging it, we won't know the position to save it